.quickstart/quickstart.yml

```diff
@@ -0,0 +1,17 @@
+database_key: amazon_selling_partner_database
+schema_key: amazon_selling_partner_schema
+
+dbt_versions: ">=1.3.0 <2.0.0"
+
+table_variables:
+  amazon_ads__portfolio_history_enabled:
+    - orders
+    - order_item
+  amazon_selling_partner__using_catalog_module:
```

Comment:
It seems there are a number of other tables that should be tied to this variable. Is there a reason they're not included here?

Reply:
I didn't include them because `item_summary` is the core table of the catalog module, and if someone were to not have `item_image` or `item_dimension` or something like that, the package's item transformations would still otherwise be valuable. That's pretty moot though since the package would therefore fail. 

I suppose I was trying to think of what we'd rather have in this case: A) the package fails for a customer and they hopefully open a ticket for us to support their use case, or B) the package doesn't run any of the item/catalog stuff -- passing but producing limited results -- and hopefully the customer opens a ticket for us to work with the catalog tables they do have.

Reply:
Reverted to the empty tmp/base model approach, so we can keep these variables tied to the core tables only